### DISCUSSION TITLE
why is there 3 icons on my mac taskbar when i open a movie in smplayer?

### DISCUSSION THREAD

**Discussion by andras-gyarmati** [smplayer-dev/smplayer#780] [Q&A]
I have a macbook pro and when I play a movie or anything three smplayer icons appear on the taskbar, one of them has a dot below it indication that it's running, the other two are just there. are they helper processes or what? i can "remove from dock" them individually and the others stay there.

**Comment by smplayer-dev** [comment]
smplayer calls mpv (or mplayer) to play the video. It also uses yt-dlp for youtube videos.
